src/Microsoft.VisualBasic/tests/VB/Dummy.vb

```diff
@@ -8,14 +8,14 @@ Imports Xunit
 
 Namespace Microsoft.VisualBasic.Tests
 
-    Public Class Dummy
+    Public NotInheritable Class Dummy
 
         <Fact>
-        Public Sub Dummy()
+        Public Shared Sub Dummy()
 
-            Dim dateTimeNowBefore As DateTime = DateTime.Now()
```

Comment:
why are we changing this file ?

Reply:
This file can be deleted it was a dummy test to show what a test looked like